### ISSUE TITLE
backwards compatibility for OpenTelemetry

### ISSUE THREAD

**Issue by michaelli321** [census-instrumentation/opencensus-specs#272]
Is there any plan on making OpenCensus compatible with OpenTelemetry?

**Comment by mtwo** [comment]
Yes, though we don't have a formal design ready. We (Google) have a new member joining our team in a few weeks who will be picking this up.
